_maps/map_files/cyberiad/z3.dmm

```diff
@@ -1,357 +1,234 @@
-"az" = (/obj/effect/decal/warning_stripes/northeastcorner,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aA" = (/obj/effect/decal/warning_stripes/northwest,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aB" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4; level = 1},/obj/effect/decal/warning_stripes/west,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aC" = (/obj/effect/decal/warning_stripes/southwestcorner,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aD" = (/obj/structure/closet/crate,/obj/machinery/light_switch{pixel_y = -28},/obj/item/clothing/glasses/night,/obj/effect/decal/warning_stripes/southwest,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aE" = (/obj/structure/closet/crate,/obj/item/aicard,/obj/item/multitool,/obj/machinery/camera{c_tag = "Telecomms Entrance South"; dir = 1; network = list("Telecomms")},/obj/effect/decal/warning_stripes/southwest,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aF" = (/obj/effect/decal/warning_stripes/south,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
-"aG" = (/obj/machinery/light{dir = 4},/obj/structure/closet/emcloset,/obj/effect/decal/warning_stripes/southeast,/turf/simulated/floor/plasteel,/area/tcommsat/entrance)
+"ae" = (/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"; tag = ""},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"af" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"ag" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/obj/machinery/light,/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"ah" = (/obj/structure/cable{d1 = 1; d2 = 4; icon_state = "1-4"; tag = ""},/obj/structure/cable{d1 = 1; d2 = 8; icon_state = "1-8"},/obj/machinery/light,/obj/machinery/camera{c_tag = "Research E.X.P.E.R.I-MENTOR Lab"; dir = 1; network = list("Research","SS13")},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"ai" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"aj" = (/obj/structure/cable{d1 = 2; d2 = 8; icon_state = "2-8"; tag = ""},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"ak" = (/obj/structure/cable{d1 = 1; d2 = 2; icon_state = "1-2"},/obj/machinery/light{dir = 4},/obj/machinery/camera{c_tag = "Labor Camp Storage"; dir = 8; network = list("Labor")},/turf/simulated/floor/plating/airless/catwalk,/area/tcommsat/powercontrol)
+"al" = (/turf/simulated/wall/r_wall,/area/tcommsat/server)
```

Comment:
As said earlier 1 layer of r-walls is not good enough. I like the idea however that the computer can be easily accessed - but the machines should be protected properly.

Reply:
I will make it 3 walls thick

Reply:
IMO, 3 walls are too much. Can't we compromise on 2?